spec/factories.rb

```diff
@@ -1,5 +1,37 @@
 FactoryBot.define do
```

Comment:
It's better to split Factories by modules.
Create a factory file for each model